addons/sale_timesheet/tests/test_sale_service.py

```diff
@@ -598,8 +598,8 @@ def test_several_uom_sol_to_planned_hours(self):
         planned_hours_for_uom = {
             'day': 8.0,
             'hour': 1.0,
-            'unit': 1.0,
-            'gram': 0.0,
+            'unit': 8.0,
+            'gram': 0.008,
```

Comment:
It is weird to convert a gram to hours/days. Are you sure that we want to do this? 🤔 😅

Reply:
@xavierbol , it's weird 🥲  but Yes, Same as 1 unit = 8 hours and 1 dozen = 96 hours as per conversion on the project's allocated hours. Thanks

Reply:
I asked to @aju-odoo and we conclude that we will keep the implementation done for the tasks and the fix should be done for the field in project.

That is, we will only convert `unit` and the working time as uom.
The other uom should be ignore (or equal to 0).

Also, I see this fix is made in master, the fix should not be in saas-15.4 since the freeze was done? 🤔 🙂